authors/apps/articles/models.py

```diff
@@ -88,6 +88,10 @@ class Comment(models.Model):
     )
     article = models.ForeignKey(Article, related_name='comments', on_delete=models.CASCADE)
     author = models.ForeignKey(Profile, related_name='comments', on_delete=models.CASCADE)
+    # The beginning of the highlight
+    begin = models.IntegerField(default=0)
```

Comment:
Where is the migration file for this? Kindly go through the slack channel conversation